main.js

```diff
@@ -52,10 +52,40 @@ function RoundToHundredths(x) {
   return Math.floor(x * 100 + 0.5) / 100;
 }
 
-function HandleNode(svgNode, scaleX, scaleY, translateX, translateY) {
+// |fillString| is expected to be "#RRGGBB" or "#RGB"].
+function PathColorFromFill(fillString) {
+  if (fillString.length === 4) {
```

Comment:
I just noticed that `fill="none"` is valid SVG, in fact I see it in a lot of the material design icons such as [this one](https://material.io/tools/icons/?icon=change_history&style=baseline). Would be nice to handle that.

Reply:
I believe that's handled so that a color path command won't be issued with:
````
const fill = svgElement.getAttribute('fill');
if (fill && fill !== 'none') {
```

Reply:
I see, so it is. Can we skip the entire path in that case?

Reply:
Do you mean skipping the output of NEW_PATH when there's a fill === "none"?

Would that run into the same issue where this svg,
```
<svg viewBox="0 0 100 100" xmlns="http://www.w3.org/2000/svg">
  <circle cx="40" cy="40" r="25" fill="green"/>
  <circle cx="60" cy="60" r="25" fill="none"/>
</svg>
```
would have the fill of the first circle bleed into the second?

Reply:
I meant skipping all output for <circle cx="60" cy="60" r="25" fill="none"/>

Reply:
Up to you - I think it makes the most sense to preserve the data from the path and have the corresponding non-colored shape in the output .icon. That way if you wanted to go in and manually hard code the color or have the color inherit from the vector icon creation in Chromium, you could.

I think there's a case for either keeping or removing the path - I'll let you tip the scales :)

Reply:
I've seen a lot of SVGs where the tool generated an invisible square the same size as the canvas. See https://chromium.googlesource.com/chromium/src/+/master/components/vector_icons/README.md#troubleshooting-icon-generation

Reply:
Seems to be handled in
```
      case 'path':
        // If fill is none, this is probably one of those worthless paths
        // of the form <path fill="none" d="M0 0h24v24H0z"/>
        if (svgElement.getAttribute('fill') == 'none')
          break;
```
For atleast the 'path' node. I have pulled this up higher to check for all nodes whether the fill == "none"